javaparser-core-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/LexicalPreservingPrinterTest.java

```diff
@@ -1104,4 +1106,16 @@ public void addedLineCommentsPrinted() {
                           "}" + EOL +
                           "}", LexicalPreservingPrinter.print(cu));
     }
+
+    @Test
+    public void issue1321() {
+        CompilationUnit compilationUnit = JavaParser.parse("class X { X() {} private void testme() {} }");
+        LexicalPreservingPrinter.setup(compilationUnit);
+
+        ClassOrInterfaceDeclaration type = compilationUnit.getClassByName("X").get();
+        type.getConstructors().get(0).setBody(new BlockStmt().addStatement("testme();"));
+
+        assertEquals("class X { X() {\n    testme();\n} private void testme() {} }", LexicalPreservingPrinter.print(compilationUnit));
```

Comment:
AppVeyor does its job: this test fails on Windows. Try `assertEqualsNoEol` or whatever it is called.

Reply:
Right, it slipped in, sorry about that